test/distributed/cases/ddl/alter_table_check_foreign_key.result

```diff
@@ -312,7 +312,7 @@ no    product_kind    product_id    customer_id    order_date
 3    20    3    3    2016-12-06 15:43:26
 4    30    4    3    2016-12-31 15:43:26
 INSERT INTO product_order VALUES (4, 30, 4, 3, '2016-12-31 15:43:26');
-Duplicate entry '4' for key 'no'
+Duplicate entry '3a15033a1504' for key '__mo_index_idx_col'
```

Comment:
Need to undo this BVT change. This occurs because PreInsert Phase throws PK error before the ActualInsert. I need to understand where UK handles CheckDuplicates.